recipes/mcxtrace-suite/meta.yaml

```diff
@@ -0,0 +1,134 @@
+{% set version = "3.4.9" %}
+{% set sha_value = "ccab20cca6e842d5b643ae30c0cd3330c3af31375b205f251cfafc086fd8000b" %}
+#NB get sha256 with: curl -sL <URL> | openssl sha256
+
+package:
+  name: mcxtrace-suite
+  version: {{ version }}
+
+source:
+  url: https://github.com/McStasMcXtrace/McCode/archive/mcxtrace-{{ version }}.tar.gz
+  sha256: {{ sha_value }}
+  folder: src
+
+build:
+  number: 2
+
+outputs:
+
+  - name: mcxtrace-core
+    script: install-files-core.sh  # [not win]
+    script: install-files-core.bat  # [win]
+    build:  # [win]
+      skip: true  # [win]
```

Comment:
If you are skipping Windows do it globally and remove all the Windows specific stuff from the recipe. This is a bit confusing.

Reply:
I've taken almost everything Windows related out now - except for the "data" sub package which at the moment is a simple xcopy in the batch file.
https://github.com/conda-forge/staged-recipes/pull/25503/commits/ca1baf72015a7c33be1446f6cd5bdbfad4ff0c4f